tactics/pfedit.mli

```diff
@@ -65,7 +65,7 @@ val build_constant_by_tactic
   :  name:Id.t
   -> ?opaque:Proof_global.opacity_flag
   -> uctx:UState.t
-  -> named_context_val
+  -> sign:named_context_val
```

Comment:
Not sure about the value of this change but won't oppose

Reply:
It was used in a few other places, it was just to label all the "accesory" args here.